#566 Android: Main Screen Added

javaman97 opened the pull request:
HomeScreen has been Added 

https://github.com/codervivek5/VigyBag/assets/75328768/678fd315-eb9c-4169-969a-75a38d04fced

**Majorly Used Concepts:-**
1) Nested RecyclerView (Parent & Child)
2) Library Used for Rating Bar 
https://github.com/zhanghai/MaterialRatingBar
3) Material Design for CardViews
4) Followed Single Activity Architecture
5) Data/Model Class

javaman97 commented:
@07jasjeet Please have a look and let me know if further changes required

07jasjeet commented on Android/app/src/main/java/com/vigbag/android/ui/home/HomeFragment.kt:
```diff
@@ -0,0 +1,72 @@
+    }
+
+    private fun onaddAllData() {
+        headphoneList=ArrayList()
```
Every assignment should have appropriate spacing.
For example, here it should be `headphoneList = ArrayList()`. Spacing is very important in order to make your code look nice. Please change this everywhere because I can see it at a lot of places. After that I think we should be good to go ahead and merge your work.

javaman97 replied:
@07jasjeet  Sure I will add Spacing in assignment

codervivek5 commented:
Kindly make some changes  in Top UI between the search bar and the headphones.....remove image and only use text 
![image](https://github.com/codervivek5/VigyBag/assets/62784060/28dbc268-d30c-45bb-9561-41da0ea0c937)

javaman97 commented:
@codervivek5 Alright,I will remove the shopping cart  image and use only Text

javaman97 commented:
![New Main Screen](https://github.com/codervivek5/VigyBag/assets/75328768/20a7fa94-315b-47c9-b623-36e83c10979c)

@codervivek5 Made the changes with Top UI, removed the shopping kart image and added only Texts as suggested.
Please have a look!

07jasjeet commented:
Nice work LGTM. @codervivek5 shall we merge?

codervivek5 commented:
> Nice work LGTM. @codervivek5 shall we merge?

sure .....you can merge it
